examples/v1beta1/taskruns/alpha/object-param-result.yaml

```diff
@@ -26,10 +26,22 @@ spec:
       - name: echo-object-params
```

Comment:
suggest renaming the step to `validate-object-params`

Reply:
updated